Production/python/Production.py

```diff
@@ -69,16 +74,22 @@
 if options.eventList != '':
     process.source.eventsToProcess = cms.untracked.VEventRange(re.split(',', options.eventList))
 
-import RecoTauTag.RecoTau.tools.runTauIdMVA as tauIdConfig
+tauAtMiniTools.addTauReReco(process)
+tauAtMiniTools.adaptTauToMiniAODReReco(process, options.reclusterJets)
+
+process.combinatoricRecoTaus.builders[0].signalConeSize = cms.string('max(min(0.2, 4.528/(pt()^0.8982)), 0.03)') ## change to quantile 0.95
+process.selectedPatTaus.cut = cms.string('pt > 18.')   ## remove DMFinding filter (was pt > 18. && tauID(\'decayModeFindingNewDMs\')> 0.5)
+
+tauIdConfig = importlib.import_module('TauMLTools.Production.runTauIdMVA')
 updatedTauName = "slimmedTausNewID"
 tauIdEmbedder = tauIdConfig.TauIDEmbedder(
     process, cms, debug = False, updatedTauName = updatedTauName,
-    toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2", "deepTau2017v2p1", "deepTau2017v2p1", "againstEle2018",  ]
+    toKeep = [ "2017v2", "dR0p32017v2", "newDM2017v2", "deepTau2017v2p1"]
 )
-tauIdEmbedder.runTauID()
+tauIdEmbedder.runTauID(tau_collection = 'selectedPatTaus')
 tauSrc_InputTag = cms.InputTag('slimmedTausNewID')
 
-tauJetdR = 0.2
+tauJetdR = 0.8
```

Comment:
should be 0.3: 0.8 - 0.5

Reply:
fixed, see https://github.com/cms-tau-pog/TauMLTools/pull/13/commits/eb0c5b9a8440bb74fa54d2eeef676b0d38489d9e